app/Models/FeedDAO.php

```diff
@@ -374,8 +374,7 @@ public function listFeedsNewestItemUsec(?int $id_feed = null): array {
 	 * @return array<int,FreshRSS_Feed> where the key is the feed ID
 	 */
 	public function listFeedsOrderUpdate(int $defaultCacheDuration = 3600, int $limit = 0): array {
-		$sql = 'SELECT id, url, kind, category, name, website, `lastUpdate`, `pathEntries`, `httpAuth`, ttl, attributes, `cache_nbEntries`, `cache_nbUnreads` '
```

Comment:
`description` was missing in the list, and we have some logic depending on `description` being blank or not